prometheus_es_exporter/__init__.py

```diff
@@ -370,6 +373,7 @@ def main():
 
         config = configparser.ConfigParser()
         config.read_file(open(args.config_file))
+        config.read(glob.glob('{}/*.cfg'.format(args.config_dir)))
```

Comment:
Would be good to sort the results of `glob.glob()` so we can guarantee the order the files are merged, in case a section occurs in multiple files.